server-pro/startup.sh

```diff
@@ -24,6 +24,15 @@ deactivate() {
 }
 trap deactivate EXIT
 
+# Call verify installation command and write to log file
+verify_installation () {
+   echo "==VERIFY INSTALLATION==";
+   mkdir -p $DIAGNOSTIC_DIR
+   chmod -R 777 $DIAGNOSTIC_DIR
```

Comment:
Let's not use `-R` here - that's a bit destructive, potentially 😄 Just dropping the `-R` should be sufficient!
```suggestion
   chmod 777 $DIAGNOSTIC_DIR
```